### PR TITLE
show hostname in triggers

### PR DESCRIPTION
**Pull request by aaboonstra**
Add GetHostName() function to Trigger which will return the first hostname (and in general the only) attached to the trigger. Print the hostname to stdout by using the trigger.GetHostName function rather than the empty Hostname field.

Closes kovetskiy/zabbixctl#9

### CODE REVIEW

File: trigger.go [hunk 1]
```diff
@@ -26,6 +30,10 @@ func (trigger *Trigger) String() string {
 		trigger.Hostname + " " + trigger.Description
 }
 
+func (trigger *Trigger) GetHostName() string {
+	return trigger.Hosts[0].Name
```
Comment (kovetskiy) [inline trigger.go]: can you please add check that len(trigger.Hosts) > 0?

**Comment by kovetskiy** [comment]
Hi! Thanks for the contribution!

**Comment by aaboonstra** [comment]
Ah for sure, do you rather have an empty string or a "missing hostname" as
fallback?

Gr,
Alex

2017-10-31 13:56 GMT+01:00 Egor Kovetskiy <notifications@github.com>:

**Comment by kovetskiy** [comment]
Yeah, it will be nice, we can return `<missing>` value in that case.

**Comment by kovetskiy** [comment]
Merged, thanks!

**Pull request merged**